hash_practice/exercises.py

```diff
@@ -22,8 +57,40 @@ def valid_sudoku(table):
         Each element can either be a ".", or a digit 1-9
         The same digit cannot appear twice or more in the same 
         row, column or 3x3 subgrid
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n^2)
+        Space Complexity: O(n)

```

Comment:
👍 Just note since sudoku boards are always 9x9, the time and space complexities don't really apply because they never scale up or down.